### PR TITLE
Allow specifying schema in redshift methods

### PR DESCRIPTION
**Pull request by agclever**
Going forward we want to store data from different teams/sources in different schemas on redshift. See https://clever.atlassian.net/browse/INFRA-667 for more details

Tested: Unit tests.

### CODE REVIEW

File: redshift/redshift.go [hunk 1]
```diff
@@ -64,9 +64,9 @@ func (r *Redshift) logAndExec(cmd string, creds bool) (sql.Result, error) {
 }
 
 // CopyJSONDataFromS3 copies JSON data present in an S3 file into a redshift table.
-func (r *Redshift) CopyJSONDataFromS3(table, file, jsonpathsFile, awsRegion string) error {
+func (r *Redshift) CopyJSONDataFromS3(schema, table, file, jsonpathsFile, awsRegion string) error {
 	copyCmd := fmt.Sprint(
-		"COPY ", table, " FROM '", file, "' WITH",
+		"COPY \"", schema, "\".\"", table, "\" FROM '", file, "' WITH",
```
Comment (nathanleiby) [inline redshift/redshift.go]: Might be more readable with backticks for string literal:
```
`COPY "`
```

Comment (nathanleiby) [inline redshift/redshift.go]: This comment applies throughout

**Comment by nathanleiby** [comment]
Suggest updating "running" section in README

**Comment by nathanleiby** [comment]
lgtm

File: redshift/redshift.go (hunk 1 above)
Comment (agclever) [inline redshift/redshift.go]: Done.

**Comment by agclever** [comment]
Updated README. Merging after tests pass.

**Pull request merged**